### PR TITLE
Dominick

### PR DESCRIPTION
**Pull request by PaperPony**

### CODE REVIEW

**Review by feneel — COMMENTED** [review]
Make the changes you think are required and then raise the PR

File: data/petsData.js [hunk 1]
```diff
@@ -1,11 +1,318 @@
...
+ * At least one of the given criteria must be specified.
+ * 
+ * @param {*} city : The city to search fors
+ * @param {*} state : The state to search for
+ * @param {*} zip : The zip to search for
+ * @param {*} petType : The petType to search for
+ * 
+ * @return Returns an array of pets that match the given criteria.
+ */
+async function homePageSearch(city, state, zip, petType) {
```
[the remark this answers is not in the document]
Comment (PaperPony) [reply data/petsData.js:21]: I'm not sure what you mean by this?

File: data/petsData.js [hunk 2]
```diff
@@ -1,15 +1,333 @@
+const { ObjectId } = require('mongodb');
```
Comment (rtnkumar) [inline data/petsData.js:1]: remove this { ObjectId }

File: tasks/awesomeTextFile.txt [hunk 3]
```diff
@@ -0,0 +1,478 @@
+    try {
```
Comment (rtnkumar) [inline tasks/awesomeTextFile.txt:1]: delete this file

File: data/petsData.js (hunk 2 above)
Comment (PaperPony) [reply data/petsData.js:1]: We need this for ObjectId conversions when validating the existence of owners in the user collection

**Pull request merged**